array_api_tests/test_has_names.py

```diff
@@ -9,6 +9,8 @@
 from .stubs import (array_attributes, array_methods, category_to_funcs,
                     extension_to_funcs, EXTENSIONS)
 
+pytestmark = pytest.mark.ci
```

Comment:
These tests feel a bit redundant for downstream CI runs, given the primary tests cover this implicitly, so to me it just adds noise to get the same failures twice.

Reply:
If one of these tests fails that means the function or attribute doesn't exist at all. The error message here will be quite clear on that, whereas the error message in the other tests might not be as clear, especially for missing attributes. 

Also, it's relevant because I want to include these failures as part of the xfails so that I know when they are fixed.